test/spec/features/popup-menu/PopupMenuSpec.js

```diff
@@ -342,14 +363,12 @@ describe('features/popup', function() {
 
         // then
         expect(popupMenu._current).to.exist;
-        expect(popupMenu._current.headerEntries[0].id).to.eql('entryA');
-        expect(popupMenu._current.entries[0].id).to.eql('entryB');
+        expect(popupMenu._current.headerEntries['entryA']).to.exist;
```

Comment:
to.have.keys can be a good matcher to express this:

```
expect(popupMenu._current.entries).to.have.keys('entryA', 'entryB');
```

Reply:
Let's do that.